northstar/src/runtime/state.rs

```diff
@@ -667,6 +667,11 @@ impl<'a> State<'a> {
             }
         }
 
+        // Spawn mount tasks onto the executor
+        let mounts = mounts
+            .drain(..)
+            .map(|t| self.executor.spawn_with_handle(t).unwrap());
```

Comment:
why do you have to do this manually? I always thought that the API to use for spawning parallel tasks is to use `Task::spawn`

Reply:
The executor the runtime is running on is a `CurrentThreadExecutor` If we would spawn the mount futures onto that executor we wouldn't have any benefit because they're executed one by one. By spawning them onto a MT executor the blocking mount calls can be done in parallel and we get a speedup when mounting a larger amount of containers at once.

Reply:
ok...that is of course true! very good!